package/MDAnalysis/analysis/align.py

```diff
@@ -943,32 +951,33 @@ def _conclude(self):
             logging.disable(logging.NOTSET)
 
     @property
-    def universe(self):
+    def universe(self) -> Type[Universe]:
         wmsg = ("The `universe` attribute was deprecated in MDAnalysis 2.0.0 "
                 "and will be removed in MDAnalysis 3.0.0. Please use "
                 "`results.universe` instead.")
         warnings.warn(wmsg, DeprecationWarning)
         return self.results.universe
 
     @property
-    def positions(self):
+    def positions(self) -> np.ndarray:
         wmsg = ("The `positions` attribute was deprecated in MDAnalysis 2.0.0 "
                 "and will be removed in MDAnalysis 3.0.0. Please use "
                 "`results.positions` instead.")
         warnings.warn(wmsg, DeprecationWarning)
         return self.results.positions
 
     @property
-    def rmsd(self):
+    def rmsd(self) -> float:
         wmsg = ("The `rmsd` attribute was deprecated in MDAnalysis 2.0.0 "
                 "and will be removed in MDAnalysis 3.0.0. Please use "
                 "`results.rmsd` instead.")
         warnings.warn(wmsg, DeprecationWarning)
         return self.results.rmsd
 
 
-def sequence_alignment(mobile, reference, match_score=2, mismatch_penalty=-1,
-                       gap_penalty=-2, gapextension_penalty=-0.1):
+def sequence_alignment(mobile: Type[AtomGroup], reference: Type[AtomGroup], match_score: float = 2,
```

Comment:
Sorry if I'm just out of the loop on this--doesn't `Type[AtomGroup]` accept `type(AtomGroup)` per https://docs.python.org/3/library/typing.html#typing.Type instead of an instance? It looks like it would also accept the actual *class object* rather than instance--is that what we'd want?

Reply:
Actually , even I wasn't really sure if I should Type hint it as Atomgroup or Type[Atomgroup] 😅.

Reply:
`Type[AtomGroup]` is the `AtomGroup` class. You may need that in some places in core, but here you need an `AtomGroup` object/instance so just `AtomGroup`.